doc/source/dev/api-dev/array_api.rst

```diff
@@ -268,11 +266,9 @@ The following pytest markers are available:
 
     python dev.py test -b all -m array_api_backends
 
-  Note that this includes tests that use the ``xp`` fixture indirectly through another
-  array API fixture, such as ``@pytest.mark.usefixtures("skip_xp_backends")``, even if
-  they don't explicitly consume ``xp`` themselves.
-
 * OBSOLETE: ``array_api_compatible`` (does nothing; pending removal)
+* OBSOLETE: ``pytest.mark.usefixtures("skip_xp_backends")`` (does nothing; pending removal)
+* OBSOLETE: ``pytest.mark.usefixtures("xfail_xp_backends")`` (does nothing; pending removal)
```

Comment:
Would be nice to follow up on all these sooner than later.

Reply:
Next PR, I promise.